modules/vaos/app/services/vaos/middleware/response/errors.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module VAOS
+  module Middleware
+    module Response
+      class Errors < Faraday::Response::Middleware
+        def on_complete(env)
+          return if env.success?
+
+          caller = "VAOS.#{caller_locations(14, 1)[0].label}"
+          Raven.extra_context(message: env.body, url: env.url)
+          case env.status
+          when 403
+            raise Common::Exceptions::BackendServiceException.new(
+              'VAOS_403',
+              source: caller
+            )
+          when 500
+            raise Common::Exceptions::BackendServiceException.new(
+              'VAOS_502',
+              source: caller
```

Comment:
these are going to get pretty unruly fast... do we want a range of 500s? Also does breakers still have an opportunity to do it's thing or does this middleware circumvent that?

Reply:
it's after breakers in the chain